### PR TITLE
MS-744 Do they need help with communication? Page

### PR DESCRIPTION
**Pull request by jjrobothamho**
Do they need help with communication? Page + acceptance tests

### CODE REVIEW

File: apps/nrm/views/pv-other-help-with-communication.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
```
Comment (jjrobothamho) [inline apps/nrm/views/pv-other-help-with-communication.html]: Need this file to override the partials page from appearing outside the option as well as within the option.

File: acceptance-test/user-pathways/util/selectors.js [hunk 2]
```diff
@@ -75,4 +76,6 @@ module.exports = {
     DOES_PV_HAVE_CHILDREN_NO_OPTION,
     PV_NATIONALITY,
     INTERPRETER_NO_OPTION,
+    COMMUNICATION_AID_NO_OPTION,
+
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/util/selectors.js]: Is that an extra line?

File: apps/nrm/views/pv-other-help-with-communication.html (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/nrm/views/pv-other-help-with-communication.html]: Nice one, always good idea to comment in a PR. Might be worth having a comment like this in the file itself in the future? What do you think? No big deal in this PR

**Comment by sulthan-ahmed** [comment]
Just that line I think you need to remove

Otherwise looks good

File: apps/nrm/views/pv-other-help-with-communication.html (hunk 1 above)
Comment (jjrobothamho) [reply apps/nrm/views/pv-other-help-with-communication.html]: No worries that's an idea might make a PR for that once I have finished the form just one bit of validation left to do.

File: acceptance-test/user-pathways/util/selectors.js (hunk 2 above)
Comment (sulthan-ahmed) [reply acceptance-test/user-pathways/util/selectors.js]: is the line still there or is it just github?

Comment (sulthan-ahmed) [reply acceptance-test/user-pathways/util/selectors.js]: it's github

**Pull request merged**